cookie-policy.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="description" content="The Lagrangian Ocean Analysis team at Utrecht University uses and develops the OceanParcels framework to investigate how ocean currents move 'stuff' around">
+    <meta property="og:image" content="http://oceanparcels.org/images/parcelslogo_large.png">
+    <meta property="twitter:image" content="http://oceanparcels.org/images/parcelslogo_large.png">
+    <meta name="author" content="The OceanParcels team">
+
+    <title>Cookie policy</title>
+    <!-- Bootstrap core CSS -->
+    <link href="vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet">
+
+    <!-- Font Awesome icons -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="vendor/academicons/css/academicons.min.css"/>
+
+    <!-- syntax highlighting for Python and Bash -->
+    <link href="vendor/prism/prism.css" rel="stylesheet" />
+    <script src="vendor/prism/prism.js"></script>
+
+    <!-- Custom styles for this template -->
+    <link href="css/modern-business.css" rel="stylesheet">
+
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script type="text/javascript">
+        (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+            (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+            m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+        })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+        ga('create', 'UA-96368922-1', 'auto');
+        ga('send', 'pageview');
+    </script>
+</head>
+<body>
+
+<nav class="navbar fixed-top navbar-expand-lg navbar-dark bg-dark fixed-top">
+    <div class="container">
+        <a class="navbar-brand" href="index.html"><img src="images/parcelslogo_inverse.png" height="30px">&ensp;Ocean<b>Parcels</b></a>
+        <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+        </button>
+    </div>
+</nav>
+
+<div class="container">
+    <br>
+    <h2> Cookie policy on the OceanParcels website</h2>
+    On this website we use Google Analytics, a web analytics service offered by Google Inc. ("Google"). We do this to provide our visitors with the best possible guide to using the Parcels code. We track the navigation on this website to see which part of the documentation may need improving. We also track the location and browser of our visitors to better accommodate our audience.
```

Comment:
Since we don't repeat the word Google, not sure we need to define it
```suggestion
    On this website we use Google Analytics, a web analytics service offered by Google Inc. We do this to provide our visitors with the best possible guide to using the Parcels code. We track the navigation on this website to see which part of the documentation may need improving. We also track the location and browser of our visitors to better accommodate our audience.
```